worldedit_commands/init.lua

```diff
@@ -1528,13 +1528,13 @@ worldedit.register_command("lua", {
 		return true, param
 	end,
 	func = function(name, param)
-		local err = worldedit.lua(param)
-		if err then
-			worldedit.player_notify(name, "code error: " .. err)
-			minetest.log("action", name.." tried to execute "..param)
+		local good, ret = worldedit.lua(param, name)
+		if good then
+			worldedit.player_notify(name, "code successfully executed, returns with " .. ret, false)

```

Comment:
if ret is nil it would be better to leave off the ", returns with" part

Reply:
fixed in a23a886

Reply:
> if ret is nil it would be better to leave off the ", returns with" part

`//lua ="nil"` would still say the code returns with string "nil" however `//lua` does not mention return value (nil was returned)
`//lua =minetest.get_node_light(pos)` works as intended

Reply:
you could capture the return values in a table to allow differentiating nothing being returned and `nil` being returned

Reply:
> you could capture the return values in a table to allow differentiating nothing being returned and `nil` being returned

hello, I don't understand the motivation behing the message, I'm not sure if I need to do anything about this one.

Reply:
> you could capture the return values in a table to allow differentiating nothing being returned and `nil` being returned

![kép](https://github.com/Uberi/Minetest-WorldEdit/assets/5818404/51753948-1136-4dfc-8b24-6cfafb1df26c)

I'm happy with the current behaviour

Reply:
Me too. I thought you meant something else.

Reply:
> Me too. I thought you meant something else.

that's good to hear. In that case, is there anything else that needs to be done?